extension/content.js

```diff
@@ -26,6 +31,55 @@ function linkifyBranchRefs() {
 	});
 }
 
+function commentIsUseless(type, el) {
+	if (uselessContent[type].text.indexOf(el.innerText) >= 0) {
+		return true;
+	}
+	// check if there is exactly one child element, that has one other child node
+	// using `childNodes` because this also includes text
+	if (el.children.length === 1 && el.children[0].childNodes.length === 1) {
+		const onlyChild = el.children[0].childNodes[0];
+		if (onlyChild.tagName === 'IMG' && uselessContent[type].emoji.indexOf(onlyChild.title) >= 0) {
+			return true;
+		}
+	}
+}
+
+function renderVoteCount(type, count) {
+	const sidebar = document.querySelector('#partial-discussion-sidebar');
```

Comment:
I've included a jQuery compatible mini lib here to make DOM stuff less verbose. Can you use it?